cmake/modules/AddSwift.cmake

```diff
@@ -239,6 +239,10 @@ function(_add_variant_c_compile_flags)
     list(APPEND result "-DNDEBUG")
   endif()
 
+  if("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
+    list(APPEND result "-Wno-inconsistent-missing-override")
```

Comment:
I reverted this commit, removed the MSVC checks (as clang-cl needs them but defined `MSC_VER`). We also now ignore unknown pragmas so this is pointless.
I also added a TODO to the pragmas that I shall try to address once I'm not so busy. Thanks